tests/foreman/cli/test_activationkey.py

```diff
@@ -619,17 +620,19 @@ def test_positive_usage_limit(module_org, module_location, target_sat):
             'lifecycle-environment-id': env['id'],
             'content-view': new_cv['name'],
             'organization-id': module_org.id,
-            'max-hosts': '1',
+            'max-hosts': max_hosts,
         }
     )
-    with Broker(nick='rhel7', host_class=ContentHost, _count=2) as clients:
+    with Broker(nick='rhel8', host_class=ContentHost, _count=2) as clients:
```

Comment:
How about using `settings.content_host.default_rhel_version` here?
```suggestion
    with Broker(nick=f'rhel{settings.content_host.default_rhel_version}', host_class=ContentHost, _count=2) as clients:
```

Reply:
Probably possible but we would need to update the nick too since it is just "9" and DVC tries to deploy 9.0 for which we don't have template.

Reply:
Updated the decorator to use the default rhel ver.